lib/spark.js

```diff
@@ -100,6 +100,8 @@ export default class Spark {
     const carBuffer = new ArrayBuffer(0, { maxByteLength: MAX_CAR_SIZE })
     const carBytes = new Uint8Array(carBuffer)
 
+    stats.startAt = new Date()
+
```

Comment:
Note - other timestamps are already set within this function.
- `stats.firstByteAt` is set inside the try block below
- `stats.endAt` is set after the try/finally block